osfoffline/views/preferences.py

```diff
@@ -154,8 +155,13 @@ def selector(self, selected_index):
             user = session.query(User).filter(User.logged_in).one()
             self.preferences_window.label.setText(self._translate("Preferences", user.full_name))
 
-            self.preferences_window.treeWidget.setCursor(QtCore.Qt.BusyCursor)
-            self.node_fetcher.finished[list].connect(self.populate_item_tree)
+            self._executor = QtCore.QThread()
```

Comment:
This is what prevents the cursor from being permanently a watch.  I talked to @chrisseto about it and this is what he helped me come up with.